packages/@react-spectrum/accordion/src/Accordion.tsx

```diff
@@ -10,91 +10,106 @@
  * governing permissions and limitations under the License.
  */
 
+import {AccordionItemProps, AccordionPanelProps} from 'react-aria-components/src/Accordion';
+import {AriaLabelingProps, DOMProps, DOMRef, StyleProps} from '@react-types/shared';
+import {Button, Header, Heading, AccordionItem as RACAccordionItem, AccordionPanel as RACAccordionPanel} from 'react-aria-components';
 import ChevronLeftMedium from '@spectrum-icons/ui/ChevronLeftMedium';
 import ChevronRightMedium from '@spectrum-icons/ui/ChevronRightMedium';
 import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
-import {DOMRef, Node} from '@react-types/shared';
-import {filterDOMProps, mergeProps} from '@react-aria/utils';
-import {FocusRing} from '@react-aria/focus';
-import React, {forwardRef, useRef} from 'react';
-import {SpectrumAccordionProps} from '@react-types/accordion';
+import {filterDOMProps} from '@react-aria/utils';
+import React, {forwardRef} from 'react';
 import styles from '@adobe/spectrum-css-temp/components/accordion/vars.css';
-import {TreeState, useTreeState} from '@react-stately/tree';
-import {useAccordion, useAccordionItem} from '@react-aria/accordion';
-import {useHover} from '@react-aria/interactions';
 import {useLocale} from '@react-aria/i18n';
 import {useProviderProps} from '@react-spectrum/provider';
 
+// TODO: handle types
+export interface SpectrumAccordionGroupProps extends StyleProps, DOMProps, AriaLabelingProps {
+  children: React.ReactNode
+}
 
-function Accordion<T extends object>(props: SpectrumAccordionProps<T>, ref: DOMRef<HTMLDivElement>) {
+function AccordionGroup(props: SpectrumAccordionGroupProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
why are we keeping the AccordionGroup if it doesn't appear to *do* anything? It doesn't even have a role

Reply:
I think there is value in having it for propagating `size`, `isQuiet`, `density`, and `isDisabled` down to child items.